### PR TITLE
Update urllib3's version of install_requires to match requests

### PR DESCRIPTION
**Pull request by kizashitakata**
refs: https://github.com/psf/requests/blob/master/setup.py

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,9 +31,8 @@
     keywords=['Kaggle', 'API'],
     entry_points={'console_scripts': ['kaggle = kaggle.cli:main']},
     install_requires=[
-        # Restriction that urllib3's version is less than 1.25 needed to avoid
-        # requests dependency problem.
-        'urllib3 >= 1.21.1, < 1.25',
+        # Restriction that urllib3's version needed to avoid requests dependency problem.
+        'urllib3 >= 1.21.1, < 1.26, != 1.25.0, != 1.25.1'
```
Comment (edwardtulane) [inline setup.py]: There is a comma missing at the end of the line.

Comment (kizashitakata) [reply setup.py]: Thank you for the indication!

**Comment by kizashitakata** [comment]
@skshetry 
Thank you for your advice!
I'm sorry that I'm not good at using git ... Is this OK?

**Comment by skshetry** [comment]
@kizashitakata, That's perfect. :+1:

**Pull request closed without merging**